.github/workflows/frontend_runtime_application_pr.yml

```diff
@@ -73,17 +73,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
-# Permissions needed to push the artifact to the S3 bucket and show the status in the PR.
-permissions:
```

Comment:
We really should have never set permissions like this, but instead only updated them for each individual job as appropriate.